test_marvin_actions.py

```diff
@@ -166,6 +166,13 @@ def testVideoOfToday(self):
                 self.assertActionOutput(marvin_actions.marvinVideoOfToday, "dagens video", response)
         self.assertActionSilent(marvin_actions.marvinVideoOfToday, "videoidag")
 
+    def testVideoOfTodayFail(self):
+        """Tests that marvin says the right thing when he doesn't have a video for the day"""
+        with mock.patch("marvin_actions.getString") as gs:
+            gs.return_value = ""
```

Comment:
will getString return an empty string if the key doesn't exist?

Reply:
Nope, it throws a KeyError. Maybe the code should be wrapped in a try/catch block?

Reply:
i mostly wanted the tests to test the actual behavior, rather than force execution of all code branches